### PR TITLE
Add log passthrough for lock-exec

### PR DESCRIPTION
**Pull request by JoshStern**
Writes command output directly to std IO and no longer returns a buffer.

https://www.loom.com/share/fc29e6dce02e43059e495169468caf0e

### CODE REVIEW

**Comment by ejeun** [comment]
👀 

just curious what you'll expect to happen when this goes to prod. what's your prediction?

File: lock/run.go [hunk 1]
```diff
@@ -20,8 +23,17 @@ func (c *Client) Run(ctx context.Context, key, command string) (string, error) {
...
+
+	// Write to std outputs + combined buffer
+	var outbuf bytes.Buffer
+	cmd.Stdout = io.MultiWriter(os.Stdout, &outbuf)
+	cmd.Stderr = io.MultiWriter(os.Stderr, &outbuf)
+
+	cmderr := cmd.Run()
+	cmdout := outbuf.Bytes()
 
 	return strings.TrimSpace(string(cmdout)), cmderr
```
[the remark this answers is not in the document]
Comment (JoshStern) [reply lock/run.go:38]: I could also update that log to avoid the double-print

**Review by lingrino — APPROVED** [review]
Yay! Thank you!

**Comment by lingrino** [comment]
I will merge this and tag a new version

**Pull request merged**